tools/hcluster_sg/.shed.yml

```diff
@@ -1,11 +1,8 @@
 categories:
 - Phylogenetics
 description: Hierarchically clustering on a sparse graph
-long_description: |
-    hcluster_sg is a tool for hierarchically clustering on a sparse graph.
-    
-    https://github.com/douglasgscofield/hcluster
+long_description: hcluster_sg is a tool for hierarchically clustering on a sparse graph. https://github.com/douglasgscofield/hcluster
```

Comment:
This change is not needed, only tabs are a problem, indentation with (2 or 4) spaces is fine.